odoo/service/security.py

```diff
@@ -24,5 +17,5 @@ def check_session(session, env):
     expected = self._compute_session_token(session.sid)
     if expected and odoo.tools.misc.consteq(expected, session.session_token):
         return True
-    self._invalidate_session_cache()
+    self.clear_caches()
```

Comment:
@xmo-odoo 

@guewen @gurneyalex @odony and I have participated in the following thread:
 - https://twitter.com/guewenb/status/1278946522068123650?s=21

The global cache is cleared in all workers for all models.
Is not this line execute too frequently?

Reply:
@moylop260 it should be OK: this should only be reached when a user has a session which still exists but whose token is invalid, and once `check)sessuib` returns `False` the session is immediately destroyed and this is not even reachable (with the same session-id). Though it's possible that the cache-clearing is unnecessary regardless, we need to check why it was adde with the original author and (eventually) whether that's still needed.